packages/backend/server/src/config/affine.self.ts

```diff
@@ -94,5 +94,7 @@ if (AFFiNE.deploy) {
 if (AFFiNE.flavor.doc) {
   AFFiNE.docService.useRpc = false;
 } else {
+  // TODO(@fengmk2): merge this config after CLOUD-116 is done
   AFFiNE.docService.useRpc = true;
+  AFFiNE.doc.manager.enableUpdateAutoMerging = false;
```

Comment:
could be removed and always enabled if `DocSerivce` get initialized(doc service will and other service will not)

Reply:
https://github.com/toeverything/AFFiNE/blob/canary/packages/backend/server/src/data/index.ts#L8

The data CLI app will manually disable the automation merge capability, only judging `AFFiNE.flavor.doc`, which may have potential issues.